portfolio/src/main/webapp/index.html

```diff
@@ -105,50 +116,124 @@ <h2>arrington's details</h2>
-            <p>These are my projects.</p>
-            <p>At some point, more will be added here.</p>
+            <h3>arrington's projects and coursework</h3>
+            <h4>my projects</h4>
+            <table>
+                <tr>
+                    <th>search</th>
+                    <td>
+                        implemented the pagerank algorithm to build a local 
+                        search engine (indexer + querier) with scala.
+                    </td>
+                </tr>
+                <tr>
+                    <th>game</th>
+                    <td>
+                        implemented a minimax algorithm to build an AI for a connect 4
+                        game with reasonML in VS code.
+                    </td>
+                </tr>
+                <tr>
+                    <th>rackette</th>
+                    <td>
+                        built a parser and interpreter for the programming langauge
+                        racket in reasonML with VS code.
+                    </td>
+                </tr>
+            </table>
+            <h4>my relevant coursework</h4>
+            <table>
+                <tr>
+                    <th>spring + summer 2020</th>
+                    <td>discrete math and probability, statistical inference, japanese 200</td>
+                </tr>
+                <tr>
+                    <th>fall 2020</th>
+                    <td>calculus II, linear algebra, japanese 100</td>
+                </tr>
+            </table>
+        </section>
+
+        <section id="comments">
+          <h3>arrington's friends comments</h3>
+          <p>displaying <span class="num">4</span> of <span class="num">12345</span> comments</p>
+          <table>
+                <tr>
+                    <th>charlie brown <img class="center" src="images/charlie.jpg" width="200" height="200"></th>
+                    <td>arrington is super cool and the most amazing person on the planet.</td>
+                </tr>
+                <tr>
+                    <th>woodstock <img class="center" src="images/woodstock.jpg" width="200" height="200"></th>
+                    <td>*bird noises*</td>
+                </tr>
+                <tr>
+                    <th>lucy <img class="center" src="images/lucy.jpg" width="200" height="200"></th>
+                    <td>i'll hold the ball and you kick it. it's thanksgiving</td>
+                </tr>
+                <tr>
+                    <th>
+                        charlie brown's mom <img class="center" src="images/not-found.png" width="200" height="200">
+                        </th>
+                    <td>
+                        wah wah wah wah wah waah wah wah. wah wah waaah wah wah wah wah waah wahh wah. 
+                        wah wah wah wah wah waah wah wah waaah wah wah. wah wah wah waah wah wah wahwah waaah wah wah wah.
+                         wah waah wah.
+                         </td>
+                </tr>
+            </table>
```

Comment:
Something cool to try when you reach the backend side of the projects would be to actually get comments from aa server and create a way to add more comments. Then you can dynamically create the table rows.